sungc/deglint.py

```diff
@@ -85,15 +86,15 @@ def deglint_ofile(self, spatialRes, out_dir, visible_bfile):
 
     def check_sensor(self):
         if (
-            (not self.sensor == "sentinel-2a")
-            and (not self.sensor == "sentinel-2b")
+            (not self.sensor == "sentinel_2a")
+            and (not self.sensor == "sentinel_2b")
             and (not self.sensor == "landsat-5")
             and (not self.sensor == "landsat-7")
             and (not self.sensor == "landsat-8")
             and (not self.sensor == "worldview-2")
         ):
             raise Exception(
-                'sensor must be "sentinel-2", "landsat-5",'
+                'sensor must be "sentinel_2", "landsat-5",'
```

Comment:
A defined list might be easier to handle rather than code the evaluation directly. Well cleaner at least.
e.g.

```python
SUPPORTED SENSORS = [
    "sentinel_2a",
    "sentinel_2b",
    "landsat-5",
    # ...
]

if sensor not in SUPPORTED_SENSORS:
    msg = f"Supported sensors are: {SUPPORTED_SENSORS}, recieved {sensor}"
    raise Exception(msg)
```

Reply:
Thanks, much neater indeed. I'll make this change